scripts/start_ngi_reports.sh

```diff
@@ -2,23 +2,8 @@
 # NGI Reports startup
 #
 # Adds the scripts directory to the user's PATH so that ngi_reports
-# can be executed from the command line easily. Also a helper bash
-# function to run pandoc easily.
+# can be executed from the command line easily.
 #
-# Note that this requires pandoc to be installed separately.
-# If you're running on Linux with RPM x86_64 you can use the
-# uppmax_ngi_reports.sh script instead, which runs the bundled
-# pandoc binary.
 
 ngi_reports_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd ../ && pwd )
```

Comment:
I think we can just delete this whole file, no? Just give the package a command line interface. No need to mess around with paths and Bash and stuff.

Reply:
Sure, I can add it in irma-provision.

Reply:
Why do we need an alias? If it has a command in `setup.py` it should just work once installed, no?

Reply:
Yes, its just that we use the `make_report` command a lot to regenerate reports from the md file. So it could be a bit faster to have an alias for `ngi_reports project_summary --markdown_file` for regeneration. We would call `ngi_reports` in the normal case, of course.

Reply:
`ngi_reports proj_summary --md`? 😅